integration-tests/plugins/__tests__/inventory/products/get-product.js

```diff
@@ -0,0 +1,109 @@
+const path = require("path")
```

Comment:
question: All the integration tests files seems to already exists but they are all marked as new and without the original tests, am i missing something?

Reply:
@pKorsholm

Reply:
They don't already exist, I've added to test suites to validate the expand params since we didn't have dedicated test-suites for those

Reply:
I must have still had them because I swithed my bad 😂